crates/tests/databases-tests/src/cockroach/snapshots/databases_tests__cockroach__schema_tests__schema_test__get_schema.snap

```diff
@@ -381,6 +417,18 @@ expression: result
             "type": "named",
             "name": "int2"
           }
+        },
+        "repair_catalog_corruption": {
```

Comment:
I think there are other functions here that should be filtered, `repair_catalog_corruption`, `pg_has_role`, `is_constraint_active`, `set_trace_verbose` are a few names that pop up.